curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-an-export-fallback-with-export-default.english.md

```diff
@@ -23,7 +23,7 @@ The following function should be the fallback value for the module. Please add t
 
 ```yml
 tests:
-  - text: Proper used of <code>export</code> fallback.
+  - text: Proper use of <code>export</code> fallback.
     testString: getUserInput => assert(getUserInput('index').match(/export\s+default\s+function\s+subtract\(x,y\)\s+{return\s+x\s-\s+y;}/g), 'Proper used of <code>export</code> fallback.');
```

Comment:
Could you also change the typo in the `testString`

```suggestion
     testString: getUserInput => assert(getUserInput('index').match(/export\s+default\s+function\s+subtract\(x,y\)\s+{return\s+x\s-\s+y;}/g), 'Proper use of <code>export</code> fallback.');
```

Reply:
Have added a new commit, should be completed, sorry about the miss.